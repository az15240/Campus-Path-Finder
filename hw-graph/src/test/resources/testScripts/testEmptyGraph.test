# Test creating many new empty graphs, ('A'), ('B'), and ('C').

CreateGraph A
ListNodes A

CreateGraph B
ListNodes B

CreateGraph C
ListNodes C