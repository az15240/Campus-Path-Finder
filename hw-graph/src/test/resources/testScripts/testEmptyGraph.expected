# Test creating many new empty graphs, ('A'), ('B'), and ('C').

created graph A
A contains:

created graph B
B contains:

created graph C
C contains: